docs/tools/terraform/get-started.rst

```diff
@@ -0,0 +1,126 @@
+For more details, refer to the `Project members and roles page <https://developer.aiven.io/docs/platform/concepts/projects_accounts_access.html#project-members-and-roles>`_.
+
+``provider.tf`` file:
+
+.. code:: bash
+
+   terraform {
+      required_providers {
+         aiven = {
+            source  = "aiven/aiven"
+            version = ">= 2.6.0, < 3.0.0"
+         }
+      }
+   }
+
+   provider "aiven" {
+      api_token = var.aiven_api_token
+   }
+
+
+3. Avoid including sensitive values in configuration files that are under source control. Rather than using sensitive information within this *variables.tf* file, you can use a ``*.tfvars`` file so that Terraform receives the values during runtime.
+
+``variables.tf`` file:
+
+.. code:: bash
+
+   variable "aiven_api_token" {
+      description = "Aiven console API token"
+      type = string
+   }
+
+   variable "project_name" {
+      description = "Aiven console project name"
+      type        = string
+   }
+
+
+4. This is where you put the actual values for Aiven API token and Aiven console project name. This file is passed to Terraform using the ``-var-file=`` flag.
+
+``var-values.tfvars`` file:
+
+.. code:: bash
+
+   aiven_api_token = "<YOUR-AIVEN-AUTHENTICATION-TOKEN-GOES-HERE>"
+   project_name = "<YOUR-AIVEN-CONSOLE-PROJECT-NAME-GOES-HERE>"
+
+
+Execution
+'''''''''
+
+Create an empty folder and add the above files to that folder. Then execute the following commands in order:
+
+.. code:: bash
+
+   terraform init 
+
+This command performs several different initialization steps in order to prepare the current working directory for use with Terraform. In our case, this command automatically finds, downloads, and installs the necessary Aiven Terraform Provider plugins.
+
+.. code:: bash
+
+   terraform plan -var-file=var-values.tfvars
+
+This command creates an execution plan and shows you the resources that will be created (or modified) for you. This command does not actually create any resource; this is more like a preview.
+
+.. code:: bash
+
+   terraform apply -var-file=var-values.tfvars
+
+If you're satisfied with ``terraform plan``, you execute ``terraform apply`` command which actually does the task or creating (or modifying) your infrastructure resources. 
+
+
+Clean up
+''''''''
+
+If this was a test environment, be sure to delete the resources once you're done to avoid consuming unwanted bills. To be confident about the service termination, you can create a speculative destroy plan by running the following command:
+
+.. code:: bash
+
+   terraform plan -destroy
+
+This will run ``terraform plan`` in destroy mode and show you the proposed destroy changes without executing them.
+
+.. warning::
+
+   Use the following command with caution. This will actually delete resources that might have important data.
+
+.. code:: bash
+
+   terraform destroy -var-file=var-values.tfvars
+
+
+Further reference
```

Comment:
We could also reference the cookbook either here or as a note near the `services.tf` example.

Reply:
Added a reference to the cookbook near the `services.tf` section.